packages/components/src/FilterBar/FilterBar.tsx

```diff
@@ -0,0 +1,36 @@
+import React from "react"
+import classnames from "classnames"
+import { OverrideClassName } from "~types/OverrideClassName"
+import {
+  FilterBarProvider,
+  FilterBarProviderProps,
+} from "./context/FilterBarContext"
+import { FiltersValues } from "./context/types"
+import { FilterBarSelect } from "./subcomponents/FilterSelect/FilterSelect"
+import styles from "./FilterBar.module.scss"
+
+export type FilterBarProps<ValuesMap extends FiltersValues> = OverrideClassName<
+  Omit<FilterBarProviderProps<ValuesMap>, "children">
+>
+
+export const FilterBar = <ValuesMap extends FiltersValues>({
+  filters,
+  classNameOverride,
+  ...providerProps
+}: FilterBarProps<ValuesMap>): JSX.Element => (
+  <FilterBarProvider<ValuesMap> filters={filters} {...providerProps}>
+    {(activeFilters): JSX.Element => (
+      <div className={classnames(styles.filterBar, classNameOverride)}>
+        {Object.values(activeFilters).map(({ id, Component }) => (
+          <React.Fragment key={id}>
```

Comment:
I know we expect all the `id` given to be unique, but should we trust that? Would adding a short UUID on the end mess thing up?

Reply:
We should trust it. The `id` is used for more than just this key after all.